spec/spec_helper.rb

```diff
@@ -87,6 +87,9 @@ def handle_current_dir_change
   c.expect_with :rspec do |expectations|
     expectations.include_chain_clauses_in_custom_matcher_descriptions = true
     expectations.max_formatted_output_length = 1000
+    if expectations.respond_to? :strict_predicate_matchers=
+      expectations.strict_predicate_matchers = true
+    end
```

Comment:
I guess we can skip that, and add separately when https://github.com/rspec/rspec-expectations/pull/1196 is merged. Otherwise, we'll have to remove the check later on anyway.

Reply:
It's a chicken and egg problem. Without this,  rspec/rspec-expectations#1196 will break the build as it introduces changes in the failure messages in this spec suite.